pkg/security/probe/probe_ebpf.go

```diff
@@ -196,9 +196,19 @@ func (p *EBPFProbe) selectFentryMode() {
 
 	supported := p.kernelVersion.HaveFentrySupport()
 	if !supported {
```

Comment:
nit: `if !p.kernelVersion.HaveFentrySupport() {` instead ?

Reply:
yes I can clean this up definitely